### ISSUE TITLE
QAT of resnet50 failured when running  on GPU GTX 1080

### ISSUE THREAD

**Issue by dongdongzi** [NVIDIA/TensorRT#2988]
## Description
I tried to run QAT of resnet50 on GTX1080, but with the increase of training epoches, the accuracy of the model decreases, Why?

## Environment

**TensorRT Version**:8.4.3

**NVIDIA GPU**:GTX 1080

**NVIDIA Driver Version**: 470.141.03 

**CUDA Version**:11.4

**CUDNN Version**:7.5.1

Operating System:ubuntu server 18.04

Python Version (if applicable):3.6

PyTorch Version (if applicable):1.7.0

```[tasklist]
### Tasks
```

```[tasklist]
### Tasks
```

**Comment by zerollzeng** [comment]
Can you provide a reproduce for this issue? also + @ttyio for viz

**Comment by dongdongzi** [comment]
I have solved this problem, just adjust shuffle is True in torch.utils.data.DataLoader.
